code/obj/flock/structure/flock_structure_ghost.dm

```diff
@@ -28,6 +28,12 @@
 		flock_speak(null, "ERROR: No Structure Tealprint Assigned, Deleting", flock)
 		qdel(src) //no exist if building null
 
+/obj/flock_structure/ghost/Click(location, control, params)
+	if ("alt" in params2list(params) || !istype(usr, /mob/living/intangible/flock/flockmind))
```

Comment:
Still appears if flockdrone controller is a flockmind, which can be annoying if the flockmind itself is trying to put stuff into a building

Reply:
TIL `in` binds like that, fixed.